cg/meta/upload/mutant/mutant.py

```diff
@@ -0,0 +1,18 @@
+from click import Context
+
+from cg.meta.upload.upload_api import UploadAPI
+from cg.meta.workflow.mutant import MutantAnalysisAPI
+from cg.models.cg_config import CGConfig
+from cg.store.models import Analysis, Case
+
+
+class MutantUploadAPI(UploadAPI):
+
+    def __init__(self, config: CGConfig):
+        self.analysis_api: MutantAnalysisAPI = MutantAnalysisAPI(config)
+        super().__init__(config=config, analysis_api=self.analysis_api)
+
+    def upload(self, ctx: Context, case: Case, restart: bool) -> None:
```

Comment:
I just had a look at the balsamic upload api since you patched it.
It has a call like so:

```Python
  if UploadGenotypesAPI.is_suitable_for_genotype_upload(case):
            ctx.invoke(upload_genotypes, family_id=case.internal_id, re_upload=restart)
```

It seems like the upload apis are calling CLI functions within them. 
Seems like some technical debt that we should address at some point

Reply:
yeah, saw that too. I would definitely want to change that but it is out of scope for this PR. Is it worse to change the signature for one Upload API than to propagate the signature?

Reply:
I think the Upload API is riddled with technical debt and warrants a re-work in a project fashion. I would make an issue and label it as technical debt that we can use to setup a project.